runtime/runtime_test.go

```diff
@@ -917,33 +917,6 @@ func TestRuntimeTransactionWithArguments(t *testing.T) {
 			},
 			expectedLogs: []string{"42"},
 		},
-		{
```

Comment:
Why was this not valid before? 🤔

Reply:
Oh this is still invalid. However, this now returns an error at the value conversion step (even before args are validated for types), because we now cannot insert the invalid element into the dictionary anymore. 

But the catch is, when I implemented it, the value-conversion used to panic (didn't return errors), and therefore cannot assert it here like the rest of the tests. So I removed it from here, since it is already covered in the below tests:  https://github.com/onflow/cadence/pull/1103/files#diff-863660c79fae3d9b381c2dff3025cd88dc46eb62571b754e7695acbfeab4fbfeR2186

Reply:
Oh, I totally misread the code. The script is fine, it is the argument / value that is invalid!

Also, thank you for the explanation why it got removed, that makes sense 👍